projects/knora-ui/src/lib/viewer/values/date-value/date-value.component.html

```diff
@@ -1,19 +1,24 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-      <kui-date-input #dateInput [formControlName]="'dateValue'" class="value" placeholder="Date value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-date-input>
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-        </textarea>
-    </mat-form-field>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <div *ngIf="valueFormControl.value.end; else date">
+        <p class="rm-value date-start">Period Start: {{valueFormControl.value.start | knoraDate}}</p>
+        <p class="rm-value date-end">Period End: {{valueFormControl.value.end | knoraDate}}</p>
+    </div>
+    <ng-template #date>
+        <p class="rm-value">Date: {{valueFormControl.value | knoraDate}}</p>
+        <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+    </ng-template>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+        <mat-form-field class="large-field child-value-component" floatLabel="never">
+            <kui-date-input #dateInput [formControlName]="'dateValue'" class="value" placeholder="Date value"
+                [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-date-input>
```

Comment:
I think the readonly checks are obsolete now. Shall we remove the `readonly` `@Input` from the wrapped component?

Reply:
Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-206